example/vectors.dart

```diff
@@ -52,15 +109,26 @@ class Point {
   }
 
   Point operator +(Vector vector) {
-    Point potision = Point(0, 0);
+    Point position = Point(0, 0);
 
-    potision.x = x + vector.x;
-    potision.y = y + vector.y;
+    position.x = x + vector.x;
+    position.y = y + vector.y;
 
-    return potision;
+    return position;
   }
 }
 
+// Subtracting two points / vectors
+// return a new vector with a result of the subtract.
```

Comment:
"...with the result of the subtraction."

Reply:
It will return the result of subtraction.